tests/customerSite/menu/viewCoffee/coffeeCupHasCorrectCost.spec.js

```
@@ -1,19 +1,35 @@
 import { test } from '../../../_fixtures/fixtures';
+import { allure } from 'allure-playwright';
+
 import { priceFormatStr } from '../../../../src/common/helpers/priceFormatters';
 import { COFFEE_NAMES, COFFEE_PRICES } from '../../../../src/constants';
 
-let testParameters = [];
+const testParameters = Object.entries(COFFEE_NAMES).map(([key, name]) => ({
+  coffee: name,
+  price: COFFEE_PRICES[key],
+}));
 
-for (const [key, value] of Object.entries(COFFEE_NAMES)) {
-  testParameters.push({ coffee: value, price: COFFEE_PRICES[key] });
-}
+test.describe('Menu > View Coffee', () => {
+  // Shared Allure labels
+  test.beforeEach(async () => {
+    await allure.parentSuite('Customer site');
+    await allure.suite('Menu');
+    await allure.subSuite('View Coffee');
+    await allure.epic('CoffeeCart Customer site');
+    await allure.feature('Menu');
+    await allure.severity('minor');
+  });
 
-testParameters.forEach(({ coffee, price }) => {
-  test(`The ${coffee} cup has correct cost`, async ({ menuPage }) => {
-    const priceStr = priceFormatStr(price);
+  testParameters.forEach(({ coffee, price }) => {
+    test(`The ${coffee} cup has correct cost`, async ({ menuPage }) => {
```

This assignment replaces `testInfo.allure` with wrappers that call the module-level `allure` (a singleton). That effectively creates a fallback/override to a module-level singleton and can mask configuration problems. It directly violates: **"HIGH: Remove any fallback from testInfo.allure to a module-level singleton for attaching Allure metadata... All metadata operations must use the Playwright adapter bound to the current test."**

Instead of overriding `testInfo.allure`, rely on the Playwright-provided `testInfo.allure` and ensure it exists; do not forward to a module-level instance.